File: Difficulty: Basic/Cube root of a number/cube-root-of-a-number.java
//{ Driver Code Starts
// Initial Template for Java

import java.io.*;
import java.util.*;

class GFG {
    public static void main(String args[]) throws IOException {
        BufferedReader read = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(read.readLine());
        while (t-- > 0) {
            int N = Integer.parseInt(read.readLine());

            Solution ob = new Solution();
            System.out.println(ob.cubeRoot(N));

            System.out.println("~");
        }
    }
}
// } Driver Code Ends


// User function Template for Java

class Solution {
    static int cubeRoot(int n) {
        // code here
        int ans = (int)(Math.cbrt(n));
        return ans;
    }
};

File: Difficulty: Basic/Cube root of a number/cube-root-of-a-number.cpp
//{ Driver Code Starts
#include <bits/stdc++.h>
using namespace std;


// } Driver Code Ends

class Solution {
  public:
    int cubeRoot(int n) {
        // code here
        return cbrt(n);
    }
};


//{ Driver Code Starts.
int main() {
    int t;
    cin >> t;
    while (t--) {
        int N;

        cin >> N;

        Solution ob;
        cout << ob.cubeRoot(N) << endl;

        cout << "~"
             << "\n";
    }
    return 0;
}
// } Driver Code Ends